### ISSUE TITLE
Scope of funding section

### ISSUE THREAD

**Issue by npch** [WSSSPE/WG-Best-Practices#3]
What do we see as the scope of the section on funding?

Is it:
1. Best practice from a funder's perspective, giving an overview of the different types of funding scheme implemented and how they are assessed
2. Best practice on funding scientific software from a PI / project manager's perspective, giving an overview of the different models of funding and examples of success and failure
3. Something else?

I'm able to lead on 2), contribute to 1) but not lead, and obviously don't know what 3) is yet!

**Comment by abani1** [comment]
Neil
if you take 2 I will try 1 and hopefully Dan will step in for that one too.

abani

On Tue, May 24, 2016 at 9:09 AM, Neil Chue Hong notifications@github.com
wrote:

> What do we see as the scope of the section on funding?
> 
> Is it:
> 
>    1.
> 
>    Best practice from a funder's perspective, giving an overview of the
>    different types of funding scheme implemented and how they are assessed
>    2.
> 
>    Best practice on funding scientific software from a PI / project
>    manager's perspective, giving an overview of the different models of
>    funding and examples of success and failure
>    3.
> 
>    Something else?
> 
> I'm able to lead on 2), contribute to 1) but not lead, and obviously don't
> know what 3) is yet!
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> https://github.com/WSSSPE/WG-Best-Practices/issues/3

## 

Abani Patra

**Comment by sandragesing** [comment]
I also would like to contribute here and suggest an additional topic. 
3. One solution could be to have (similar to HPC teams, which are quite common) research programmer teams at universities, who can work on diverse projects. We have this model here at ND and the advantage is that they gain then more experience in diverse projects and can use synergistic effects for the projects when similar cyberinfrastructure is needed. It is a re-charge model - PIs don't have to employ further programmers in their team but can "rent out" manpower from these centralized teams, which could make sense for smaller projects.

**Comment by iliant** [comment]
Hi all,

I can help with 2) too.

Ilian

**Comment by iliant** [comment]
This is to acknowledge that 3)'s "rent out" concept already happens both in the EU and USA at government labs.  At DL we Advance Performance Engineering group that is rented out.  In the UK we have academic computational communities (www.ccp.ac.uk) that support work at government labs as well as universities.

Ilian

**Comment by danielskatz** [comment]
Yes, I can help with 1.  I would also like to try to help with 2.

Dan

On May 24, 2016, at 8:23 AM, abani1 <notifications@github.com<mailto:notifications@github.com>> wrote:

Neil
if you take 2 I will try 1 and hopefully Dan will step in for that one too.

abani

On Tue, May 24, 2016 at 9:09 AM, Neil Chue Hong <notifications@github.com<mailto:notifications@github.com>>
wrote:

> What do we see as the scope of the section on funding?
> 
> Is it:
> 
> 1.
> 
> Best practice from a funder's perspective, giving an overview of the
> different types of funding scheme implemented and how they are assessed
> 2.
> 
> Best practice on funding scientific software from a PI / project
> manager's perspective, giving an overview of the different models of
> funding and examples of success and failure
> 3.
> 
> Something else?
> 
> I'm able to lead on 2), contribute to 1) but not lead, and obviously don't
> know what 3) is yet!
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> https://github.com/WSSSPE/WG-Best-Practices/issues/3

## 

Abani Patra

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly or view it on GitHubhttps://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_WSSSPE_WG-2DBest-2DPractices_issues_3-23issuecomment-2D221266758&d=CwMFaQ&c=8hUWFZcy2Z-Za5rBPlktOQ&r=4d90HBeCqoFfzklYCKMj-EoIThdyUd0oF4FaGaoN8_U&m=UEJO0dBzm_nFJfsgtutRGKQuscDXwYugKTPDFSxKPsY&s=r0KMK5zUxI0tYOFKNk9V4N7DKl5sihVEvy6vGOfSV1s&e=

## 

Daniel S. Katz
Assistant Director for Scientific Software and Applications, NCSA
Research Associate Professor, ECE
Research Associate Professor, GSLIS
University of Illinois
(217) 244-8000
d.katz@ieee.orgmailto:d.katz@ieee.org or dskatz@illinois.edumailto:dskatz@illinois.edu
http://danielskatz.orghttp://danielskatz.org/
